README.md

```diff
@@ -42,6 +42,26 @@ $ cmake --build .
 This will build Fizzy as a library and since there is no public API
 (the so called *embedder API* in WebAssembly) yet, this is not very useful.
 
+## WASI
+
+Building with the `FIZZY_WASI` option will output a `fizzy-wasi` binary implementing
+the [WASI] API (a very limited subset of `wasi_snapshot_preview1`, to be precise).
```

Comment:
It's not straigtforward to find this `wasi_snapshot_preview1` in WASI repo, I think this is the right link https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/witx/wasi_snapshot_preview1.witx perhaps we should include it.

Reply:
Or this one is more readable https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/docs.md

Reply:
Added the witx because it links to the readme too.